research/arxiv_papers/CUHKSZ-TSP/EAX/compile.py

```diff
@@ -0,0 +1,65 @@
+import os
+import sys
+import shutil
+import getopt
+
+##delete and rebuild the output
+shutil.rmtree('output')
+os.mkdir('output')
+##delete and rebuild the error
+shutil.rmtree('error')
+os.mkdir('error')
+##delete EDGF
+shutil.rmtree('Sol')
+os.mkdir('Sol')
+
+os.system("rm output_all_task.log")
+os.system("rm mem0_mtrpp.sh")
+os.system("rm subtsk*")
+os.system("rm output_all*")
+
+
+def main(argv):
+	time=""
```

Comment:
Python indents four spaces. You can use black to brush it.

```shell
pip install black
black -l 120 research/arxiv_papers/CUHKSZ-TSP
```

Reply:
In my own local environment, this looks well. But in Github, it has some problems.

![image](https://github.com/user-attachments/assets/e9677e5a-4e9a-48da-b8a4-b0f1252749c3)

Reply:
Replace tab with 4 spaces

Reply:
I have replaced.